Sources/Paywalls/PaywallData+Localization.swift

```diff
@@ -15,43 +15,72 @@ import Foundation
 
 public extension PaywallData {
 
+    /// - Returns: the `Locale` being used with ``PaywallData/LocalizedConfiguration-swift.struct``
+    var locale: Locale? {
+        let singleTier = self.localizedConfiguration(for: Self.localesOrderedByPriority)?.0
+        let multiTier = self.localizedConfigurationByTier(for: Self.localesOrderedByPriority)?.0
+
+        return singleTier ?? multiTier
+    }
+
     /// - Returns: the ``PaywallData/LocalizedConfiguration-swift.struct``  to be used
-    /// based on `Locale.current` or `Locale.preferredLocales`.
+    /// based on `Locale.preferredLocales` or the default locale.
     /// -  Returns: `nil` for multi-tier paywalls.
     var localizedConfiguration: LocalizedConfiguration? {
-        return self.localizedConfiguration(for: Self.localesOrderedByPriority)
+        return self.localizedConfiguration(for: Self.localesOrderedByPriority)?.1
     }
 
     /// - Returns: the ``PaywallData/LocalizedConfiguration-swift.struct``  to be used
-    /// based on `Locale.current` or `Locale.preferredLocales`.
+    /// based on `Locale.preferredLocales` or the default locale.
     /// -  Returns: `[:]` for single-tier paywalls.
     var localizedConfigurationByTier: [String: LocalizedConfiguration]? {
-        return self.localizedConfigurationByTier(for: Self.localesOrderedByPriority)
+        return self.localizedConfigurationByTier(for: Self.localesOrderedByPriority)?.1
     }
 
     // Visible for testing
-    internal func localizedConfiguration(for preferredLocales: [Locale]) -> LocalizedConfiguration? {
+    internal func localizedConfiguration(
+        for preferredLocales: [Locale]
+    ) -> (Locale, LocalizedConfiguration)? {
```

Comment:
Consider labeling the tuple parameters so that we can refer to them by name rather than order (ie instead of `?.0`, ``?.locale`).
```suggestion
    ) -> (locale: Locale, localizedConfiguration: LocalizedConfiguration)? {
```